source/cppfront.cpp

```diff
@@ -2538,7 +2538,7 @@ class cppfront
                 }
                 printer.preempt_position(n.position());
                 emit( *type );
-                if (n.pointer_declarator) {
+                for (int i = 0; i < std::ssize(n.pointer_declarators); ++i) {
```

Comment:
Seems like a range based for would be better here so there are no index type issues.

Reply:
Actually, my first implementation was done with range-for 
```cpp
for (auto _ : n.pointer_declarators) { /**/}
```
but I was not using the argument so I decided to switch to this one.

Reply:
Maybe the `printer` is missing this functionality, or a `repeat` function is missing.

Reply:
I am trying to write consistent code with @hsutter style - there is a code with indexes so I am coping the style
```cpp
for(auto i = 0;
    i < std::ssize(prev_line_info.requests) && prev_line_info.requests[i].requested <= pos.colno;
    ++i
    )
```

but I am open to change that if needed.